docker/git-puller/bin/entrypoint.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+set -e
+
+if [[ $# -lt 1 ]]; then
+  echo "Expected 1 or more parameters but got $#."
+  cat /README.md
+  exit 1
+fi
+
+GIT_REPOS=$@
+
+# Tells git to use the provided private key and automatically (without prompt)
+# add the new hosts keys to the known hosts file
+export GIT_SSH_COMMAND="ssh -o StrictHostKeyChecking=no -i ${PRIVATE_KEY_PATH}"
```

Comment:
not sure if we should allow passing a list of `known_hosts` in here?
would be good to have @Briansbum to take a look at this and the ssh password mechanism as well